indra/newview/llmodelpreview.cpp

```diff
@@ -2523,7 +2523,7 @@ void LLModelPreview::genMeshOptimizerLODs(S32 which_lod, S32 meshopt_mode, U32 d
         out << "Invalid level of detail: " << which_lod;
         LL_WARNS() << out.str() << LL_ENDL;
         LLFloaterModelPreview::addStringToLog(out, true); // <FS:Beq/> if you don't flash the log tab on error when do you?
-        assert(lod >= -1 && lod < LLModel::NUM_LODS);
+        assert(which_lod >= -1 && which_lod < LLModel::NUM_LODS); // <FS:Ale/> use the correct variable (which_lod).
```

Comment:
Just want to see what happens if I click 'Start a review'.

Reply:
Looks like I am the one doing the "review" after that, and I have to Finish the review myself.
Look good! Though I'd use <FS:PR-Aleric/>, what you think? ;)